Avalonia.Headless.NUnit.AvaloniaTestAttribute is missing constructors from base class

**Describe the bug**
I want to use `AvaloniaTestAttribute` from `Avalonia.Headless.NUnit` to identify test cases for my NUnit tests, but it is missing the constructors from its base class ([NUnit.Framework.TestCaseAttribute](https://github.com/nunit/nunit/blob/master/src/NUnitFramework/framework/Attributes/TestCaseAttribute.cs)) so I can't pass any arguments to it.

**To Reproduce**
Attempt to write a test with `AvaloniaTestAttribute`, e.g.
```
[TestFixture]
public class Tests
{
    [AvaloniaTest(1, 2, 3)]
    public void Test(int arg1, int arg2, int expected)
    {
        int result = arg1 + arg2;
        Assert.AreEqual(expected, result);
    }
}
```
This causes `error CS1729: 'AvaloniaTestAttribute' does not contain a constructor that takes 3 arguments`.

**Expected behavior**
This code should compile and run successfully. `AvaloniaTestAttribute` should include the following constructors.
```
        public AvaloniaTestAttribute(params object?[]? arguments) : base(arguments) { }
        public AvaloniaTestAttribute(object? arg) : base(arg) { }
        public AvaloniaTestAttribute(object? arg1, object? arg2) : base(arg1, arg2) { }
```

**Desktop (please complete the following information):**

- OS: Windows 11
- Version 11.0.0-rc1.1

--- a/src/Headless/Avalonia.Headless.NUnit/AvaloniaTest.cs
+++ b/src/Headless/Avalonia.Headless.NUnit/AvaloniaTest.cs
@@ -13,7 +13,7 @@ namespace Avalonia.Headless.NUnit;
 /// such that awaited expressions resume on the test's "main thread".
 /// </summary>
 [AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = false)]
-public sealed class AvaloniaTestAttribute : TestCaseAttribute, IWrapSetUpTearDown
+public sealed class AvaloniaTestAttribute : TestAttribute, IWrapSetUpTearDown
 {
     public TestCommand Wrap(TestCommand command)
     {

--- a/tests/Avalonia.Headless.UnitTests/ThreadingTests.cs
+++ b/tests/Avalonia.Headless.UnitTests/ThreadingTests.cs
@@ -18,7 +18,7 @@ public class ThreadingTests
     }
 
 #if NUNIT
-    [AvaloniaTest(Ignore = "This test should always fail, enable to test if it fails")]
+    [AvaloniaTest, Ignore("This test should always fail, enable to test if it fails")]
 #elif XUNIT
     [AvaloniaFact(Skip = "This test should always fail, enable to test if it fails")]
 #endif

